### ISSUE TITLE
support for multiple redis servers

### ISSUE THREAD

**Issue by ddplay** [yiisoft/yii2-redis#66]

**Comment by samdark** [comment]
Yes.

**Comment by ddplay** [comment]
how to use?

**Comment by samdark** [comment]
Well, define mutiple connections then use them.

**Comment by ddplay** [comment]
well,i mean distrubite Redis usage

**Comment by samdark** [comment]
Do you mean redis cluster? http://redis.io/topics/cluster-tutorial

**Comment by crazyfd** [comment]
@samdark @cebe how to define mutiple connections in yii2 redis

**Comment by samdark** [comment]
@crazyfd multiple connections or cluster?

**Comment by crazyfd** [comment]
@samdark  redis cluster server  is work,how defind in yii2

**Comment by samdark** [comment]
If you need just multiple connections, define these as:

```php
'components' => [
    'redis1' => [
        'class' => 'yii\redis\Connection',
        'hostname' => 'localhost',
        'port' => 6379,
        'database' => 0,
    ],
    'redis2' => [
        'class' => 'yii\redis\Connection',
        'hostname' => 'localhost',
        'port' => 6380,
        'database' => 0,
    ],
]
```

**Comment by smartyPHP** [comment]
@samdark now i have tow redis server save session,i want to do that when one server is crash the another server will continue read session or write session,how to do that?

**Comment by samdark** [comment]
Via using cluster?

**Comment by smartyPHP** [comment]
@samdark yes

**Comment by acs-ferreira** [comment]
See also https://github.com/pyurin/yii2-redis-ha

**Comment by Liv1020** [comment]
@samdark How to config for redis cluster

**Comment by cebe** [comment]
@Liv1020 https://github.com/pyurin/yii2-redis-ha

**Comment by Liv1020** [comment]
@smartyPHP  https://github.com/Liv1020/yii2-redis Use Predis, and fully compatible with the yii2-redis

**Comment by cebe** [comment]
implemented by #132
